infinispan-embedded/src/test/java/org/hibernate/ogm/datastore/infinispan/test/dialect/impl/counter/TransportNotDefinedTableGenerationTest.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.infinispan.test.dialect.impl.counter;
+
+import org.hibernate.ogm.backendtck.id.TableNextValueGenerationTest;
+import org.hibernate.ogm.datastore.infinispan.InfinispanProperties;
+import org.hibernate.ogm.utils.TestForIssue;
+import org.hibernate.ogm.utils.jpa.GetterPersistenceUnitInfo;
+
+/**
+ * Repeat {@link TableNextValueGenerationTest} tests
+ * with a configuration does not defining transport
+ *
+ * @author Fabio Massimo Ercoli
+ */
+@TestForIssue(jiraKey = "OGM-1376")
+public class TransportNotDefinedTableGenerationTest extends TableNextValueGenerationTest {
```

Comment:
I'm surprised by this name. Shouldn't be LocalCachesTableGenerationTest or similar? Anyway, I will merge it and we can rename it later if needed

Reply:
I guesslocal caches don't have a transport defined, I just think it's clearer to highlight what it is present more that what it is missing.

Reply:
No biggie, anywyay